src/Core/Geographic/Coordinates.js

```diff
@@ -148,8 +150,38 @@ function _convert(coordsIn, newCrs, target) {
         }
 
         if (coordsIn.crs in proj4.defs && newCrs in proj4.defs) {
-            const p = instanceProj4(coordsIn.crs, newCrs).forward([coordsIn._values[0], coordsIn._values[1]]);
-            return target.set(newCrs, p[0], p[1], coordsIn._values[2]);
+            var val0 = coordsIn._values[0];
+            var val1 = coordsIn._values[1];
```

Comment:
Use `let` rather than `var`.

The whole function looks nicer than in your previous iterations, so good job 👍